info/available-rules.md

```diff
@@ -1,131 +1,131 @@
+| 1    | 1.6.1    | VARIABLE_NAME_INCORRECT_FORMAT            | Check: warns if the name of the variable is not in lowerCamelCase or contains non-ASCII letters<br>Fix: fixing the case format to lowerCamelCase                                                                                                                 |     | no                                                                                                                                                                                                                         | -                                                                                                                                                                                                                             |
+| 1    | 1.6.2    | FUNCTION_BOOLEAN_PREFIX                   | Check: functions/methods that return boolean should have a special prefix, such as "is/should/e.t.c"<br>Fix:                                                                                                                                                       | yes | no                                                                                                                                                                                                                         | Recursively update usages of this function in the projectAggressive fix - what if the new name will not be valid?                                                                                                                 |
+| 2    | 2.1.1    | MISSING_KDOC_TOP_LEVEL                    | Check: warns at file level internal or public class or if the function has a missing KDoc<br>Fix: no                                                                                                                                                                 | no  | no                                                                                                                                                                                                                         | Support extension for setters/gettersSupport extension for method "override"                                                                                                                                                  |
```

Comment:
"Support extension for setters/gettersSupport extension for method "override""
Please, explain.

Reply:
These are just to-do items, so let's not delve too much into this.

> Support extension for setters/getters. Support extension for overriding methods (those marked with the [`override`](https://kotlinlang.org/docs/inheritance.html#overriding-methods) keyword).

Reply:
OK